daktari/__main__.py

```diff
@@ -26,10 +26,31 @@ def main() -> int:
 
     if config.title:
         print_logo(config.title)
+
+    print_messages(config, args)
+
     all_passed = run_checks(config.checks)
     print("")
     return 0 if all_passed else 1
 
 
+def print_messages(config: Config, args):
+    if config.printLocalFileMessage:
+        print(
+            f"ⓘ  A local config file has been generated at {LOCAL_CONFIG_PATH}. "
+            f"Use this to override daktari behaviour - see the file for more details.\n"
+        )
+
+    ignored_count = len(config.ignored_checks)
+    if ignored_count > 0:
+        if args.show_ignored:
+            print("ⓘ  The following checks have been ignored:\n")
+            for check in config.ignored_checks:
+                print(f"[{check.name}]")
```

Comment:
Maybe, I don't think it's likely to be a big mystery though... and you could always tweak your local config and re-run to narrow down if you *really* had to.

I'd add it if it were trivial, but I think trying to get this extra logging would actually complicate things a fair bit (because rather than passing back a flat list of ignored_checks, we'd have to pass back something that captured more info). I'm inclined to leave it unless you feel strongly that it's worth the added complexity?

Reply:
Yeah, not worth the effort 👍